books_scraperr/spiders/books.py

```diff
@@ -0,0 +1,64 @@
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+
+from word2number import w2n
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
```

Comment:
Pls check the officials docs and follow the recommendations :

https://docs.scrapy.org/en/latest/intro/tutorial.html